index.js

```diff
@@ -138,3 +138,54 @@ newSightingForm.addEventListener('submit', function (event) {
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
 
+//**********Deliverable 1: Like an animal sighting**********
+
+const animSightings = document.querySelector('ul#animals')
+
+animSightings.addEventListener('click', function (event){
+if (event.target.matches('button.like-button')){
+    const likesDisplay = event.target.previousElementSibling
+    const animLikes = parseInt(likesDisplay.textContent) + 1
+    likesDisplay.textContent = `${animLikes} Likes`
+}
+
+//**********Deliverable 2: Delete an animal sighting**********
+
+else if (event.target.matches('button.delete-button')){
+    const animals = event.target.closest('li')
+    animals.remove()
+}
+
+//**********Deliverable 3: View the update animal sighting description form**********
+
+else if (event.target.matches('button.toggle-update-form-button')){
+    const li = event.target.closest('li')
+    const showForm = li.querySelector('form')
+
+    if(showForm.style.display === 'block'){
+        showForm.style.display = 'none'
+    }
+    else{
+        showForm.style.display = 'block'
+        }
+    }
+})
+
+
+
```

Comment:
I will leave my implementation of the bonus deliverable here.

1.  listen for any form submission that occurs within the animals UL element.
2. make sure to prevent the default action of the form.
3. find the p tag and update it's value based on the form data.
4. Don't forget to change the form field value to the new description.

Again, we can break it apart into 2 steps.

```
/***** BONUS DELIVERABLE part 1 *****/

animalsUL.addEventListener("submit", function(event){
    event.preventDefault()
    if (event.target.matches('.update-form')){
        handleUpdateFormSubmission(event)
    }
})

/***** BONUS DELIVERABLE part 2 *****/
function handleUpdateFormSubmission(e){
    e.preventDefault()
    const description = e.target.children[0].value
    const animalDescriptionPTag = e.target.parentElement.children[0]
    animalDescriptionPTag.innerText = description
    e.target.value = description
}
```